okhttp/src/commonMain/kotlin/okhttp3/Protocol.kt

```diff
@@ -15,7 +15,7 @@
  */
 package okhttp3
```

Comment:
What to do with

```
  @Deprecated("OkHttp has dropped support for SPDY. Prefer {@link #HTTP_2}.")
  SPDY_3("spdy/3.1"),
```

Reply:
It stays as is. We don't break binary compatibility.

Reply:
I'm assuming enums need to be complete with the same set for all languages, otherwise you can't define their values in commonMain.